cico_run_coverage.sh

```diff
@@ -2,9 +2,6 @@
 
 . cico_setup.sh
 
-# Use go1.11 (or newer from epel)
```

Comment:
Now I don't know why it says `from epel` here but that is not what I've written and intended.

Reply:
The comment is out of date. I'll remove it.